pkg/server/server_sql.go

```diff
@@ -610,6 +610,23 @@ func (s *sqlServer) start(
 			// present situation).
 			DistSQLMode: sessiondata.DistSQLOff,
 		})
+	{
```

Comment:
I guess I'd get this of the `migrationsExecutor`.

Reply:
I'm not sure I understand this comment.